src/libical/icaldate_p.c

```diff
@@ -0,0 +1,131 @@
+/**
+ *  SPDX-FileCopyrightText: 1986-2000, Hiram Clawson
+ *  SPDX-FileCopyrightText: 2018, Markus Minichmayr  <markus@tapkey.com>
```

Comment:
I guess this copyright notice could be removed as we don't have this throughout the library. Did I add this back then? Probably because I (probably mistakenly) assumed this was common practice back then, which it doesn't seem to be.